src/index.ts

```diff
@@ -120,6 +121,8 @@ export {
     IDataSourceProviderInjectedProps,
     IDrillableItem,
     IDrillEventExtended,
+    IDrillEvent,
+    convertDrillContextToLegacy,
```

Comment:
I don't see any tests for the conversion in current code. However, when exposing this via index, maybe we could have at least one happy-path unit test. Just optional, we can easily live without it as this is legacy and to-be-removed conversion.